src/server/qgsservercachemanager.h

```diff
@@ -0,0 +1,146 @@
+/***************************************************************************
+                        qgsservercachemanager.h
+                        -----------------------
+
+  begin                : 2018-07-05
+  copyright            : (C) 2018 by René-Luc D'Hont
+  email                : rldhont at 3liz dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSSERVERCACHEMANAGER_H
+#define QGSSERVERCACHEMANAGER_H
+
+#include "qgsservercachefilter.h"
+#include "qgsaccesscontrol.h"
+#include "qgsserverrequest.h"
+
+#include <QMultiMap>
+#include <QDomDocument>
+#include "qgsproject.h"
+#include "qgis_server.h"
+#include "qgis_sip.h"
+
+SIP_IF_MODULE( HAVE_SERVER_PYTHON_PLUGINS )
+
+class QgsServerCachePlugin;
```

Comment:
What is this class?

Reply:
It's like in `src/server/accesscontrol.h`. You will find the definition of `class QgsAccessControlPlugin;`.

And in `src/server/accesscontrolfilter.h` you'll find the preprocessor macro `QGSACCESSCONTROLPLUGIN_H` with `class SERVER_EXPORT QgsAccessControlFilter`